docs/index.html

```diff
@@ -117,3 +117,8 @@ <h4>Tree</h4>
   src="https://code.jquery.com/jquery-3.3.1.min.js"
   crossorigin="anonymous">
 </script>
+<script>
+  let select = document.getElementById('language-select')
+  select.nodeValue = 'yaml'
```

Comment:
The event is registered via `addEventListener`, so we probably need to use `dispatchEvent` to trigger it:

```suggestion
  select.value = 'yaml'
  select.dispatchEvent(new Event('change'))
```

Reply:
just change the value is enough, i have try it.

Reply:
Can you share the steps to reproduce & verify? Only changing the value does not seem to work on my machine (Chrome on macOS).